### ISSUE TITLE
[Embedded Inspector] JxBrowser's license is missing after upgraded to 71.1.6

### ISSUE THREAD

**Issue by AlexV525** [flutter/flutter-intellij#6463]
![image](https://user-images.githubusercontent.com/15884415/207621578-a088a586-896e-49ad-8b12-89b867a1db26.png)

![image](https://user-images.githubusercontent.com/15884415/207621683-a00690c3-82a7-4076-9adf-e96d53724e1e.png)

Logs from `idea.log`:
```
2022-12-14 22:23:37,714 [   7734]   INFO - #io.flutter.jxbrowser.JxBrowserManager - flutter_test_app: Unable to find JxBrowser license key file
java.io.FileNotFoundException: jxbrowser.properties file does not exist
	at io.flutter.utils.JxBrowserUtils.getJxBrowserKey(JxBrowserUtils.java:63)
	at io.flutter.jxbrowser.JxBrowserManager.setUp(JxBrowserManager.java:215)
	at io.flutter.ProjectOpenActivity.runActivity(ProjectOpenActivity.java:71)
```

**Comment by stevemessick** [comment]
@helin can you take a loot at this one?

**Comment by stevemessick** [comment]
@AlexV525 We may have to work with the JxBrowser team to get this resolved. Meanwhile, just just the non-embedded tool.

**Comment by Shvet** [comment]
Same Problem, Can you put instruction how to use other broswer in embedded tool? or it wont be possible for now?

![Screenshot_1](https://user-images.githubusercontent.com/7760064/207774898-5f708488-98a4-47ea-97ee-c416cd32d993.png)

**Comment by AlexV525** [comment]
> Same Problem, Can you put instruction how to use other broswer in embedded tool? or it wont be possible for now?

Uncheck the highlighted checkbox to use the native inspector.

![image](https://user-images.githubusercontent.com/15884415/207775614-8cb774f3-4d83-41df-9549-4b5a2b6bbb5d.png)

**Comment by AlexV525** [comment]
Instead, every action in the old inspector is not working, like widget selector mode is not triggering the select mode in Flutter apps.

**Comment by stevemessick** [comment]
This is fixed in 71.2.
